contrib/packs/actions/install.meta.yaml

```diff
@@ -17,6 +18,10 @@
       type: "string"
       default: "actions,aliases,sensors,triggers"
       description: "Possible options are all, triggers, sensors, actions, rules, aliases."
+    upgrade:
+      type: "boolean"
+      default: false
+      description: "Upgrade the pack if already installed."
```

Comment:
Hm, but this is interesting.

In future it would be good to have pack install really idempotent.
So if pack is already installed - it won't be reinstalled again, unless some special flag like that specified. Re-downloading/re-installing by default takes time and it's in fact "upgrade" or "reinstall".

This makes sense especially for configuration management tools, when you apply playbook/cookbook, which apart of deploying `st2` installs a bunch of st2 packs on the same machine. And it's normal to run same playbook on the same machine several times.